docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   storyplayer-dev:
-    image: node:16
+    image: node:20
```

Comment:
Can we specifiy this to a specific minor versions too for consistency between repos? This site seems to imply 20.9 is current LTS https://nodejs.org/en/